src/shared/GitLab/GitLabHostProvider.cs

```diff
@@ -276,18 +225,28 @@ protected override void ReleaseManagedResources()
             base.ReleaseManagedResources();
         }
 
-        private string GetRefreshTokenServiceName(InputArguments input)
+        private string GetRefreshTokenServiceName(Uri remoteUri)
```

Comment:
Is the `remoteUri` parameter being used here?

Reply:
Good spot. Fixed bug.